lib/galaxy/managers/citations.py

```diff
@@ -38,9 +38,12 @@ def _get_tool(self, tool_id):
 class DoiCache:
     def __init__(self, config):
         cache_opts = {
-            "cache.type": getattr(config, "citation_cache_type", "file"),
+            "cache.type": getattr(config, "citation_cache_type", "ext:database"),
             "cache.data_dir": getattr(config, "citation_cache_data_dir", None),
             "cache.lock_dir": getattr(config, "citation_cache_lock_dir", None),
+            "cache.url": getattr(config, "citation_cache_url", None),
+            "cache.table_name": getattr(config, "citation_cache_table_name", None),
+            "cache.schema_name": getattr(config, "citation_cache_schema_name", None),
```

Comment:
I think this should still default to file, but I also think the `getattr` isn't necessary at all and predates a refactoring of the `config` instance. All config attributes are always set now, so 

```suggestion
            "cache.type": config.citation_cache_type,
            "cache.data_dir": config.citation_cache_data_dir,
            "cache.lock_dir": config.citation_cache_lock_dir,
            "cache.url": config.citation_cache_url,
            "cache.table_name": config.citation_cache_table_name,
            "cache.schema_name": config.citation_cache_schema_name,
```
should work too